api/lib/domain/errors.js

```diff
@@ -320,6 +320,12 @@ class InvalidCertificationCandidate extends DomainError {
   }
 }
 
+class InvalidExternalUserTokenError extends DomainError {
+  constructor(message = 'L\'idToken de l\'utilisateur externe est invalide.') {
```

Comment:
On peut enlever d'échapper en mettant ‘
```suggestion
  constructor(message = 'L‘idToken de l‘utilisateur externe est invalide.') {
```

Reply:
Done